uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/annual-entitlements.html

```diff
@@ -0,0 +1,36 @@
+<div class="panel panel-default chr_annual-entitlements">
+  <div class="panel-heading">
+    <h3 class="panel-title">Annual Entitlements</h3>
+  </div>
+  <crm-loading show="entitlements.loaded.absencePeriods">
+    <div class="table-responsive" ng-show="entitlements.loaded.absencePeriods && entitlements.absencePeriods.length > 0">
```

Comment:
`entitlements.loaded.absencePeriods`check is not necessary, as when its false, the loader will be shown and this wont be visible.